saleor/settings.py

```diff
@@ -468,3 +473,15 @@ def get_host():
 DEFAULT_MENUS = {
     'top_menu_name': 'navbar',
     'bottom_menu_name': 'footer'}
+
+
+NOCAPTCHA = True
+
+ENABLE_RECAPTCHA = get_bool('ENABLE_RECAPTCHA', False)
+
```

Comment:
I'd group all those variables as they're treating same subject

Reply:
Sure! Just note that `NOCAPTCHA` is to use the new (latest) version of recaptcha (which is the "Are you humain? Just check the box.")